src/vm/builtins/builtin_runner/bitwise.zig

```diff
@@ -53,14 +93,516 @@ pub const BitwiseBuiltinRunner = struct {
         };
     }
 
+    pub fn initDefault() Self {
+        var default: bitwise_instance_def.BitwiseInstanceDef = .{};
+        return Self.init(&default, true);
```

Comment:
Maybe in the `init` function we can set `instance_def` as a pointer to constant because this is not modified and this will permit you to do

```zig
return Self.init(&bitwise_instance_def.BitwiseInstanceDef = .{}, true);
```

So I think we need to modify `init` declaration to:
```zig
pub fn init(
        instance_def: *const bitwise_instance_def.BitwiseInstanceDef,
        included: bool,
    ) Self {
```

Reply:
trying to use an assignment in passing an argument is giving me an error (feel free to sanity check me) but this works!
```
&@as(bitwise_instance_def.BitwiseInstanceDef, .{})
```